test-cases/features/per-partition-limit.yaml

```diff
@@ -21,3 +21,5 @@ instance_type_loader: 'c5.large'
 instance_type_monitor: 't2.small'
 
 user_prefix: 'per-part-limit'
+
```

Comment:
While at it, can you please change t2 to t3?

Reply:
added a new to commit, to change all t2 to t3